pubspec.yaml

```diff
@@ -17,8 +17,8 @@ dependencies:
   cupertino_icons: ^1.0.4
   wakelock: ^0.6.1+2
   meta: ^1.7.0
-  flutter_widget_from_html_core: ^0.8.5+1
-  visibility_detector: ^0.2.2
+  flutter_widget_from_html_core: ^0.8.5+3
+  visibility_detector: ^0.3.3
   path_provider: ^2.0.9
   collection: ^1.15.0
   xml: ^5.3.0
```

Comment:
upgrade xml?

Reply:
Yes, the xml dependency on line 24
`xml: ^5.3.0`

Reply:
please upgrade xml lastest version

Reply:
Updating xml should not be necessary for Flutter 3 support, so I'd argue for merging this and updating the other dependencies in a separate PR.